Apps/LabRecorder/CMakeLists.txt

```diff
@@ -58,23 +58,41 @@ add_executable(${target}
     targetver.h
 )
 
-IF(MSVC)
-    find_package(Boost REQUIRED COMPONENTS filesystem iostreams thread zlib)
-    set(zlib_libs Boost::zlib)
-ELSE()
-    find_package(Boost REQUIRED COMPONENTS filesystem iostreams thread)
-    find_package(ZLIB REQUIRED)
-    set(zlib_libs ${ZLIB_LIBRARIES})
-ENDIF()
+set(Boost_NO_SYSTEM_PATHS true)
```

Comment:
This will probably break the build for everyone with a system Boost installation

Reply:
I will make this flippable via a command line argument.

Reply:
Actually, turning this off didn't affect the outcome, so I deleted this.